packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -69,6 +69,11 @@ export const App = ({ queryParams }: AppProps) => {
   }
   const account = data!.account;
 
+  Metrics.setProperties({
+    lang: document.querySelector('html')?.getAttribute('lang'),
+    uid: account.uid,
+  });
```

Comment:
These two value could arguably also come from the get_flow process, but ideally we're not relying on it where we don't have to.